sonar-db/src/main/java/org/sonar/db/version/v53/FixMsSqlCollation.java

```diff
@@ -0,0 +1,553 @@
+      .addVarcharColumn("value_error", 64)
+      .addVarcharColumn("value_warning", 64)
+      .execute();
+
+    // properties
+    new UpdateTableCollation(context, db, "properties")
+      .addIndex("properties_key", "prop_key")
+      .addVarcharColumn("prop_key", 512)
+      .execute();
+
+    // project_links
+    new UpdateTableCollation(context, db, "project_links")
+      .addVarcharColumn("component_uuid", 50)
+      .addVarcharColumn("link_type", 20)
+      .addVarcharColumn("name", 128)
+      .addNotNullableVarcharColumn("href", 2048)
+      .execute();
+
+    // duplications_index
+    new UpdateTableCollation(context, db, "duplications_index")
+      .addIndex("duplications_index_hash", "hash")
+      .addNotNullableVarcharColumn("hash", 50)
+      .execute();
+
+    // project_measures
+    new UpdateTableCollation(context, db, "project_measures")
+      .addVarcharColumn("text_value", 4000)
+      .addVarcharColumn("alert_status", 5)
+      .addVarcharColumn("alert_text", 4000)
+      .addVarcharColumn("url", 2000)
+      .addVarcharColumn("description", 4000)
+      .execute();
+
+    // projects
+    new UpdateTableCollation(context, db, "projects")
+      .addUniqueIndex("projects_kee", "kee")
+      .addUniqueIndex("projects_uuid", "uuid")
+      .addIndex("projects_project_uuid", "project_uuid")
+      .addIndex("projects_module_uuid", "module_uuid")
+      .addIndex("projects_qualifier", "qualifier")
+      .addVarcharColumn("kee", 400)
+      .addVarcharColumn("uuid", 50)
+      .addVarcharColumn("project_uuid", 50)
+      .addVarcharColumn("module_uuid", 50)
+      .addVarcharColumn("module_uuid_path", 4000)
+      .addVarcharColumn("name", 256)
+      .addVarcharColumn("description", 2000)
+      .addVarcharColumn("scope", 3)
+      .addVarcharColumn("qualifier", 10)
+      .addVarcharColumn("deprecated_kee", 400)
+      .addVarcharColumn("path", 2000)
+      .addVarcharColumn("language", 20)
+      .addVarcharColumn("long_name", 256)
+      .execute();
+
+    // manual_measures
+    new UpdateTableCollation(context, db, "manual_measures")
+      .addIndex("manual_measures_component_uuid", "component_uuid")
+      .addVarcharColumn("component_uuid", 50)
+      .addVarcharColumn("text_value", 4000)
+      .addVarcharColumn("user_login", 255)
+      .addVarcharColumn("description", 4000)
+      .execute();
+
+    // active_rules
+    new UpdateTableCollation(context, db, "active_rules")
+      .addVarcharColumn("inheritance", 10)
+      .execute();
+
+    // user_roles
+    new UpdateTableCollation(context, db, "user_roles")
+      .addNotNullableVarcharColumn("role", 64)
+      .execute();
+
+    // active_rule_parameters
+    new UpdateTableCollation(context, db, "active_rule_parameters")
+      .addVarcharColumn("rules_parameter_key", 128)
+      .addVarcharColumn("value", 4000)
+      .execute();
+
+    // users
+    new UpdateTableCollation(context, db, "users")
+      .addUniqueIndex("users_login", "login")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "login" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)